mcnp/mcnp_input.i

```diff
@@ -308,910 +308,67 @@ C    Concrete shield
-     68164.00c -1.3587e-04
-     68166.00c -2.8780e-03
-     68167.00c -1.9764e-03
-     68168.00c -2.3455e-03
-     68170.00c -1.3117e-03
-     72174.00c -5.3358e-08
-     72176.00c -1.7743e-06
-     72177.00c -6.3099e-06
-     72178.00c -9.3069e-06
-     72179.00c -4.6728e-06
-     72180.00c -1.2103e-05
-     92234.00c -5.2292e-04
-     92235.00c -5.8755e-02
-     92236.00c -2.7027e-04
-     92238.00c -2.3797e-01
+m1190701   92235.00c -1.0000e+00                            $Fuel, 11907, Middle
 mt1190701  zr-zrh.80t h-zrh.80t
-C name: Fuel, 11907, Top
-C density = 7.2
-m1190702
-     1001.00c -1.1583e-02
-     1002.00c -2.6624e-06
-     6012.00c -3.2031e-03
-     6013.00c -3.7541e-05
-     40090.00c -3.4428e-01
-     40091.00c -7.5914e-02
-     40092.00c -1.1731e-01
-     40094.00c -1.2147e-01
-     40096.00c -1.9987e-02
-     68162.00c -1.1653e-05
-     68164.00c -1.3587e-04
-     68166.00c -2.8780e-03
-     68167.00c -1.9764e-03
-     68168.00c -2.3455e-03
-     68170.00c -1.3117e-03
-     72174.00c -5.3358e-08
-     72176.00c -1.7743e-06
-     72177.00c -6.3099e-06
-     72178.00c -9.3069e-06
-     72179.00c -4.6728e-06
-     72180.00c -1.2103e-05
-     92234.00c -5.2292e-04
-     92235.00c -5.8755e-02
-     92236.00c -2.7027e-04
-     92238.00c -2.3797e-01
+m1190702   92235.00c -1.0000e+00                               $Fuel, 11907, Top
 mt1190702  zr-zrh.80t h-zrh.80t
-C name: Steel, Stainless 304
-C density = 8.0
-m1190799
-     6012.00c -3.9536e-04
-     6013.00c -4.6337e-06
-     14028.00c -4.5933e-03
-     14029.00c -2.4168e-04
-     14030.00c -1.6499e-04
-     15031.00c -2.3000e-04
-     16032.00c -1.4207e-04
-     16033.00c -1.1568e-06
-     16034.00c -6.7533e-06
-     16036.00c -1.6825e-08
-     24050.00c -7.9299e-03
-     24052.00c -1.5903e-01
-     24053.00c -1.8380e-02
-     24054.00c -4.6613e-03
-     25055.00c -9.9999e-03
-     26054.00c -3.9616e-02
-     26056.00c -6.4489e-01
-     26057.00c -1.5160e-02
-     26058.00c -2.0529e-03
-     28058.00c -6.2157e-02
-     28060.00c -2.4767e-02
-     28061.00c -1.0946e-03
-     28062.00c -3.5473e-03
-     28064.00c -9.3243e-04
+m1190799   26056.00c -1.0000e+00                           $Steel, Stainless 304
 C
-C    General materials  
-C name: Zirconium
-C density = 6.5
-m20000
-     40090.00c -5.0704e-01
-     40091.00c -1.1180e-01
-     40092.00c -1.7277e-01
-     40094.00c -1.7890e-01
-     40096.00c -2.9436e-02
-     72174.00c -7.8583e-08
-     72176.00c -2.6131e-06
-     72177.00c -9.2930e-06
-     72178.00c -1.3707e-05
-     72179.00c -6.8819e-06
-     72180.00c -1.7824e-05
-C name: Water, Liquid
-C density = 1.0
-m30000
-     1001.00c -1.1187e-01
-     1002.00c -2.5713e-05
-     8016.00c -8.8570e-01
-     8017.00c -3.5857e-04
-     8018.00c -2.0482e-03
+C    General materials
+m20000     40092.00c -1.0000e+00                                      $Zirconium
+m30000     1001.00c  -1.0000e+00                                  $Water, Liquid
 mt30000    h-h2o.80t
```

Comment:
not sure if using `h-h2o`  is equivalent to what I am using --- we are assuming that the hydrogen is not bound in a molecule which will impact the thermal scattering cross section. I guess there is another `S(a, b)`  id for hydrogen only!

Reply:
I see. Then my question is:
shall we keep the material as it is now and use H-only thermal scattering cross section,
or change the material to be H2O and use thermal data for H2O?

For MCNP, ENDF-B/8.0 is used. Maybe it needs to be 7.1 for consistency among inputs?

Reply:
- Let's use h2o and yes please if it is possible use 7.1 since I can't use the latest version.